### DISCUSSION TITLE
SvelteKit monorepo - Error: No Output Directory named "build" found after the Build completed.

### DISCUSSION THREAD

**Discussion by gyurielf** [vercel/community#3258] [Help]
### Summary

I spent a lot of time to solve this issue but it looks like a bug instead of my fault.

I wanted to deploy a site from a sveltekit monorepo.
https://github.com/gyurielf/mono-sti/

But I struggled on the output settings. So I tried several settings to be able to deploy the site.

SvelteKit with adapter static adapter's build goes to a `build` folder by default.

So I set the output folder on the vercel control panel to:
`/apps/site/build`

But I got the following error message:
`Error: No Output Directory named "build" found after the Build completed.`

I went to investigate the logs of the deployment build, and found this:
``Wrote site to ".vercel/output/static"`` << It should be ``Wrote site to "build".``

First I tried to create a [vercel.json](https://vercel.com/docs/concepts/projects/project-configuration#outputdirectory) file on the root , I hoped that might help.
It didn't. Then I moved to the `/app/site` folder but nothing changed.

TLDR:
Users must set the `adapter-static` output to `build` otherwise it wont work.
`Wrote site to "build".`

Fun fact, the default output directory is [build](https://github.com/sveltejs/kit/blob/092edf216593a7bc5e8a6c4c395426ec2b9c1f0e/packages/adapter-static/index.js#L54) for `the adapter-static`.
You can see in the documentation as well: https://kit.svelte.dev/docs/adapter-static#usage

So here is what solved the issue:
```
/** @type {import('@sveltejs/kit').Config} */
export default {
	preprocess: [vitePreprocess()],
	kit: {
		adapter: staticAdapter({
			pages: 'build',
			assets: 'build'
		})
	}
};
```

### Example

https://github.com/gyurielf/mono-sti/

### Steps to Reproduce

- Create a sveltekit monorepo without turborepo.
- Use adapter static
- Try to deploy the app with the default sveltekit build output.
